packages/material-react-table/src/table/MRT_TableRoot.tsx

```diff
@@ -218,6 +218,7 @@ export const MRT_TableRoot: any = <TData extends Record<string, any> = {}>(
       props.localization,
       props.positionActionsColumn,
       props.renderDetailPanel,
+      props.renderRowActions,
```

Comment:
I'm wondering if we should add all the renderXXX options as dependencies to the memo ...